packages/swingset-theme-hashicorp/src/components/side-nav/index.tsx

```diff
@@ -1,20 +1,49 @@
 import { NavigationTree } from 'swingset/types'
 import Category from './category'
+import { Link } from '../link'
+import { cx } from 'class-variance-authority'
+import { ToggleButton } from './toggle-button'
 
 type SideNavBarProps = {
   categories: NavigationTree
+  isOpen: boolean
+  toggle: () => void
 }
 
-function SideNavigation({ categories }: SideNavBarProps) {
-  
+function SideNavigation({ categories, isOpen, toggle }: SideNavBarProps) {
+  const renderedCategories = categories.map((category) => (
+    <Category
+      title={category.title}
+      key={category.title}
+      items={category.children}
+    />
+  ))
 
-  const renderCategories = categories.map(
-    (category) => (
-      <Category title={category.title} key={category.title} items={category.children} />
-    )
-  )
+  return (
+    <>
+      <aside
+        aria-hidden={!isOpen}
+        className={cx(
+          'ss-hidden lg:ss-fixed lg:ss-inset-y-0 lg:ss-z-50 lg:ss-flex lg:ss-flex-col lg:ss-w-72 ss-transition-transform',
+          !isOpen && 'ss--translate-x-full ss-invisible'
```

Comment:
Added `aria-hidden` and added `ss-invisible` to fully remove element from browser accessibility tree